net-p2p/bitcoind/bitcoind-0.16.0.ebuild

```diff
@@ -0,0 +1,165 @@
+	zeromq? ( net-libs/zeromq )
+	virtual/bitcoin-leveldb
+"
+DEPEND="${RDEPEND}"
+
+DOCS=( doc/bips.md doc/files.md doc/reduce-traffic.md doc/release-notes.md )
+
+S="${WORKDIR}/${MyPN}-${BITCOINCORE_COMMITHASH}"
+
+pkg_pretend() {
+	if use knots; then
+		einfo "You are building ${PN} from Bitcoin Knots."
+		einfo "For more information, see ${KNOTS_DESC}"
+	else
+		einfo "You are building ${PN} from Bitcoin Core."
+		einfo "For more information, see ${CORE_DESC}"
+	fi
+	if use bitcoin_policy_rbf; then
+		einfo "Replace By Fee policy is enabled: Your node will preferentially mine and relay transactions paying the highest fee, regardless of receive order."
+	else
+		einfo "Replace By Fee policy is disabled: Your node will only accept the first transaction seen consuming a conflicting input, regardless of fee offered by later ones."
+	fi
+}
+
+pkg_setup() {
+	enewgroup bitcoin
+	enewuser bitcoin -1 -1 /var/lib/bitcoin bitcoin
+}
+
+KNOTS_PATCH() { echo "${WORKDIR}/${KNOTS_P}.patches/${KNOTS_P}.$@.patch"; }
+
+src_prepare() {
+	sed -i 's/runscript/openrc-run/' contrib/init/${PN}.openrc || die
+
+	sed -i 's/^\(complete -F _bitcoind bitcoind\) bitcoin-qt$/\1/' contrib/${PN}.bash-completion || die
+
+	eapply "$(KNOTS_PATCH syslibs)"
+
+	if use knots; then
+		eapply "$(KNOTS_PATCH f)"
+		eapply "$(KNOTS_PATCH branding)"
+		eapply "$(KNOTS_PATCH ts)"
+	fi
+
+	eapply_user
+
+	if ! use bitcoin_policy_rbf; then
+		sed -i 's/\(DEFAULT_ENABLE_REPLACEMENT = \)true/\1false/' src/validation.h || die
+	fi
+
+	echo '#!/bin/true' >share/genbuild.sh || die
+	mkdir -p src/obj || die
+	echo "#define BUILD_SUFFIX gentoo${PVR#${PV}}" >src/obj/build.h || die
+
+	eautoreconf
+	rm -r src/leveldb src/secp256k1 || die
+}
+
+src_configure() {
+	local my_econf=(
+		$(use_enable asm)
+		--without-qtdbus
+		--with-libevent
+		--without-qrencode
+		$(use_with upnp miniupnpc)
+		$(use_enable upnp upnp-default)
+		$(use_enable test tests)
+		$(use_enable wallet)
+		$(use_enable zeromq zmq)
+		--with-daemon
+		--disable-util-cli
+		--disable-util-tx
+		--disable-bench
+		--without-libs
+		--without-gui
+		--disable-ccache
+		--disable-static
+		--with-system-leveldb
+		--with-system-libsecp256k1
+		--with-system-univalue
+	)
+	econf "${my_econf[@]}"
+}
+
+src_install() {
+	default
+
+	rm -f "${ED%/}/usr/bin/test_bitcoin" || die
+
+	insinto /etc/bitcoin
+	newins "${FILESDIR}/bitcoin.conf" bitcoin.conf
+	fowners bitcoin:bitcoin /etc/bitcoin/bitcoin.conf
+	fperms 600 /etc/bitcoin/bitcoin.conf
+
+	newconfd "contrib/init/bitcoind.openrcconf" ${PN}
+	newinitd "contrib/init/bitcoind.openrc" ${PN}
+	systemd_newunit "${FILESDIR}/bitcoind.service-r1" "bitcoind.service"
+
+	keepdir /var/lib/bitcoin/.bitcoin
+	fperms 700 /var/lib/bitcoin
+	fowners bitcoin:bitcoin /var/lib/bitcoin/
+	fowners bitcoin:bitcoin /var/lib/bitcoin/.bitcoin
+	dosym ../../../../etc/bitcoin/bitcoin.conf /var/lib/bitcoin/.bitcoin/bitcoin.conf
+
+	dodoc doc/REST-interface.md doc/tor.md
```

Comment:
Add this to `DOCS` array? It really makes no sense to install some of the files via `DOCS`, and then add some more via `dodoc`.